src/web/js/trove/chart-lib.js

```diff
@@ -530,21 +531,34 @@
           // The only way to hijack marker events is to walk the DOM here
           // If Google changes the DOM, these lines will likely break
           const svgRoot = chart.container.querySelector('svg');
-          const slices = [...svgRoot.children].slice(2, -1);
+
+          // The order of SVG slices is *not* the order of the rows in the table!!
+          // 1 or 2 slices: drawn in reverse order
+          // >2 slices: the first row in the table is the 
+          let slices;
+          if(table.length <= 2) {
+            slices = [...svgRoot.children].slice(2, -1).reverse();
```

Comment:
More efficient to call `Array.prototype.slice.call(svgRoot.children, 2, -1)`, and avoid the extra `[...svgRoot.children]` array creation.

Reply:
good call! fixed.